cmd/tracee-ebpf/flags/filter.go

```diff
@@ -276,6 +277,11 @@ func PrepareFilter(filtersArr []string) (tracee.Filter, error) {
 			filter.Follow = true
 			continue
 		}
+
+		if strings.HasPrefix("self", filterFlag) {
```

Comment:
We also have the `set` filter check which uses a prefix so I think this should come before that one and have a full equality check.